### DISCUSSION TITLE
How to identify Postal with HTTP endpoint routing ?

### DISCUSSION THREAD

**Discussion by dtrckd** [postalserver/postal#2063] [Help with using Postal] [answered]
I see there is a token in the JSON payload, but it is different for each request....
I found no way to set a secret/token to the postal request to http endpoints for server authorization.
Is it possible to do so ?

**Comment by willpower232** [comment, marked as answer]
The request your server receives should be signed as you can see here 

https://github.com/postalserver/postal/blob/2e7aaaba5a9a37f32bdff9a9fde0379834b2a4db/lib/postal/http.rb#L40-L43

You can have a look at #432 if you want to verify the request within your application.

**Reply by dtrckd** [reply]
Noted, thanks for this.
